src/components/RestaurantCard.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const RestaurantCard = ({ restaurant }) => (
+  <div>
```

Comment:
To improve our semantic, I'd use [`article`](https://www.w3schools.com/tags/tag_article.asp) as the root element rather than `div`. I still have some questions about each tag introduced by HTML5, but I truly believe that `article` is the perfect element for this case. What do you think?